runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,183 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
Is the full PATH here needed?

Reply:
`PATH=/usr/local/bin` is really all we need. I will go ahead and update